go/vt/servenv/servenv.go

```diff
@@ -58,6 +58,12 @@ var (
 	onTermTimeout  = flag.Duration("onterm_timeout", 10*time.Second, "wait no more than this for OnTermSync handlers before stopping")
 	memProfileRate = flag.Int("mem-profile-rate", 512*1024, "profile every n bytes allocated")
 
+	// RedactDebugUIQueries controls whether full queries and bind variables are suppressed from debug UIs.
+	RedactDebugUIQueries = flag.Bool("redact-debug-ui-queries", false, "redact full queries and bind variables from debug UI")
+
+	// QueryLogFormat controls the format of the query log (either text or json)
+	QueryLogFormat = flag.String("querylog-format", "text", "format for query logs (\"text\" or \"json\")")
+
```

Comment:
What's the motivation behind defining these flags here?

servenv is a pretty generic framework. Many servers we build (or may build in the future) don't do anything with queries. So, I don't think it's a good idea to define these flags at this level.

Reply:
@sougou I rebased this on top of master and made this change to move the flags into `streamlog` and not the generic `servenv`.